addons/point_of_sale/models/pos_session.py

```diff
@@ -426,7 +426,7 @@ def _accumulate_amounts(self, data):
                     # the respective positive (negative) balance in session's invoice receivable lines. That's why the comparator to
                     # calculate `key`s for `invoice_receivable_lines` in `_create_invoice_receivable_lines` is inverted.
                     # The key ensures we don't reconcile invoice receivable lines with opposite sign together
-                    key = (order.partner_id.commercial_partner_id.id, order._get_amount_receivable() < 0, move_line.account_id.id)
+                    key = (order.partner_id.commercial_partner_id.id, move_line.account_id.id)
```

Comment:
One last thing, kindly remove the comment for this line because makes no sense anymore.